Newsletters: add 317 (2024-08-23)

harding opened the pull request:
- [ ] Lede, releases/RCs, topic entries @harding 
- [x] Client/Services section @bitschmidty 
- [x] Notable merges @Gustavojfe

bitschmidty commented:
Smaller Client/Services section this month, suggestions welcome

harding commented:
Added a news item since we didn't have any.  It's for a discussion we normally would've covered next week (it started after this PR was initially opened).

bitschmidty reviewed (commented):
News item looks good to me

LarryRuane commented on _posts/en/newsletters/2024-08-23-newsletter.md:
```suggestion
  channels] that has an exponential cost per additional bit, via grinding
```

harding replied:
Leaving as-is.  I think this is slightly more invasive a change, for no significant improvement in comprehensibility, than I'd like to make to a quote.

LarryRuane replied:
Agreed, sorry, I once again failed to notice that this is inside a quote.

LarryRuane commented on _posts/en/newsletters/2024-08-23-newsletter.md:
```suggestion
  good, but it's only a two-move protocol, so many places that wouldn't
```

LarryRuane commented on _posts/en/newsletters/2024-08-23-newsletter.md:
```suggestion
  best possible anti-exfiltration with single-round signing.
```

LarryRuane commented on _posts/en/newsletters/2024-08-23-newsletter.md:
```suggestion
  the performance of Stratum v1 and Stratum v2 protocols in different mining scenarios.
```

Gustavojfe commented:
just pushed all merge summaries for this week

harding commented:
Made all edits (or left a comment), thanks @bitschmidty and @LarryRuane!  Reviewed @bitschmidty and @Gustavojfe sections, great stuff!  Added lede, checked for updated releases/RCs (none), and added topic entries.  Thanks everyone!

LarryRuane commented on _posts/en/newsletters/2024-08-23-newsletter.md:
```suggestion
  Discussion is ongoing at the time of writing.
```

LarryRuane commented on _posts/en/newsletters/2024-08-23-newsletter.md:
```suggestion
  represents the probability that a randomly chosen channel in the network
  can forward at least 1 msat. The default value is set to 0.98, but this
```

LarryRuane commented on _posts/en/newsletters/2024-08-23-newsletter.md:
```suggestion
  other reasons: high base fee, low capacity, and high latency. In addition, a new
```

LarryRuane reviewed (commented):
A few small items... otherwise LGTM

bitschmidty commented:
Took a suggestion from @LarryRuane, squashed, and merged, thank you all! 🚀
